tap_snapchat_ads/streams.py

```diff
@@ -215,6 +226,41 @@ def remove_hours_local(self, dttm, timezone):
             hour=0, minute=0, second=0, microsecond=0).astimezone(pytz.timezone('UTC')).strftime('%Y-%m-%dT%H:%M:%SZ')
         return new_dttm
 
+    @staticmethod
+    def extract_selected_profile_data(config, client, stream_name, parent_id=None):
```

Comment:
Can you please add some code comments to this function? Also, you can add unit test cases for this function.

Reply:
yeah will add and push the changes

Reply:
added  couple of unit tests to handle edge cases